test/torch_np/numpy_tests/lib/test_function_base.py

```diff
@@ -2537,7 +2597,19 @@ def test_error_not_1d(self, vals):
             np.bincount(vals)
 
 
-@xfail  # (reason="TODO: implement")
+parametrize_interp_sc = parametrize(
+    "sc",
+    [
+        subtest(lambda x: np.float_(x), name="real"),
+        subtest(lambda x: _make_complex(x, 0), name="complex-real"),
+        subtest(lambda x: _make_complex(0, x), name="complex-imag"),
+        subtest(lambda x: _make_complex(x, np.multiply(x, -2)), name="complex-both"),
+    ],
+)
+
+
+@xpassIfTorchDynamo  # (reason="TODO: implement")
+@instantiate_parametrized_tests
```

Comment:
how come the script missed these? Could we fix the script to detect these?

Reply:
This was hiding in the false negatives output of the script, I guess.